web_src/css/repository.css

```diff
@@ -333,7 +333,7 @@
   margin-right: 3px;
 }
 
-.repository.file.list #repo-files-table td > * {
+.repository.file.list #repo-files-table td>* {
```

Comment:
Too many unrelated changes in this file.

Reply:
Sorry, I have restored some unexpected css format now